core/src/autogluon/core/pseudolabeling/pseudolabeling.py

```diff
@@ -9,12 +9,14 @@
 
 def sample_bins_uniformly(y_pred_proba: pd.DataFrame, df_indexes):
     """
-    Takes predictive probs and finds the minimum class count then samples that
-    from every class with rows with index in df_indexes
+    Takes predictive probabilities from y_pred_proba and finds the minimum
+    class count then samples minimum class count from every class with
```

Comment:
definition of "minimum class count" could be clarified

Reply:
added more exposition